### ISSUE TITLE
Testnet error: "INVALID_CONFIG: hardforks differs from embedded hardforks"

### ISSUE THREAD

**Issue by leoenes** [toncenter/ton-http-api#74]
Hello,

After following these steps, I could't communicate to testnet:

1. Download the testnet file;

`mkdir private && curl -sL https://ton-blockchain.github.io/testnet-global.config.json > private/testnet.json`

2. custom .env (basically, changing the TON_API_TONLIB_LITESERVER_CONFIG path)

> TON_API_CACHE_ENABLED=0
> TON_API_CACHE_REDIS_ENDPOINT=cache_redis
> TON_API_CACHE_REDIS_PORT=6379
> TON_API_CACHE_REDIS_TIMEOUT=1
> TON_API_LOGS_JSONIFY=0
> TON_API_LOGS_LEVEL=ERROR
> TON_API_GET_METHODS_ENABLED=1
> TON_API_HTTP_PORT=8082
> TON_API_JSON_RPC_ENABLED=1
> TON_API_ROOT_PATH=/
> TON_API_WEBSERVERS_WORKERS=1
> TON_API_TONLIB_LITESERVER_CONFIG=private/testnet.json
> TON_API_TONLIB_KEYSTORE=/tmp/ton_keystore/
> TON_API_TONLIB_PARALLEL_REQUESTS_PER_LITESERVER=50
> TON_API_TONLIB_CDLL_PATH=
> TON_API_TONLIB_REQUEST_TIMEOUT=10
> TON_API_GUNICORN_FLAGS=
> COMPOSE_FILE=docker-compose.yaml

3. Running using docker compose:

`docker compose build --no-cache && docker compose up`

4. Output Error:

![image](https://user-images.githubusercontent.com/15250987/210785178-3c6b881e-d1cf-4e0e-8d2a-df93c5691d60.png)

After that I tried changing the liteserver entry to point just to my own lite server, but I received the same error. The same steps with mainnet worked. Could you help me with this error?

**Comment by ayrat555** [comment]
@tolya-yanot @kdimentionaltree @dungeon-master-666 
hey. is it possible to take a look at this issue? :pray:

**Comment by uv-orbs** [comment]
Same here, same error

**Comment by uv-orbs** [comment]
```bash
--
50 | export TON_API_LITESERVER_CONFIG=private/testnet.json
51 | ./configure.py
52 | ```

TON_API_LITESERVER_CONFIG is misleading in the README
It has no reference in any code and it must be a confusion with  TON_API_TONLIB_LITESERVER_CONFIG

**Comment by dungeon-master-666** [comment]
Fixed with #76.
Thanks for reporting.

**Comment by leoenes** [comment]
Thank you very much!
